Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -772,8 +772,8 @@ open class LineChartRenderer: LineRadarRenderer
         
         for high in indices
         {
-            guard let set = lineData[high.dataSetIndex] as? LineChartDataSetProtocol
-                , set.isHighlightEnabled
+            guard let set = lineData[high.dataSetIndex] as? LineChartDataSetProtocol,
```

Comment:
not sure the change here. the diff shows one space is deleted?

Reply:
There was extra whitespace